project_app/pages/Reviews.tsx

```diff
@@ -73,7 +73,7 @@ export default function Reviews() {
       <ReviewBox>
          <button>Create a Review</button> 
```

Comment:
pls chang filename to revews.tsx

Reply:
All filenames are lowercase 
<img width="253" alt="Screen Shot 2023-05-23 at 4 57 52 PM" src="https://github.com/idiglove/oku-no-kumo-fe/assets/34128735/3b9ae810-d1df-41a9-ab22-2a74125c7919">